src/test/addresses_wallets.sol

```diff
@@ -96,5 +94,18 @@ contract Wallets {
         addr["BIBTA_WALLET"]                 = 0x173d85CD1754daD73cfc673944D9C8BF11A01D3F;
         addr["MIP65_WALLET"]                 = 0x29408abeCe474C85a12ce15B05efBB6A1e8587fe;
         addr["BLOCKTOWER_WALLET"]            = 0x117786ad59BC2f13cf25B2359eAa521acB0aDCD9;
+
+        // Responsible Facilitators
+        addr["GOV_ALPHA"]                    = 0x01D26f8c5cC009868A4BF66E268c17B057fF7A73;
+        addr["TECH_WALLET"]                  = 0x2dC0420A736D1F40893B9481D8968E4D7424bC0B;
+        addr["STEAKHOUSE"]                   = 0xf737C76D2B358619f7ef696cf3F94548fEcec379;
+        addr["BA_LABS"]                      = 0xDfe08A40054685E205Ed527014899d1EDe49B892;
+
+        // Ecosystem Actors
+        addr["PHOENIX_LABS_STREAM"]          = 0x115F76A98C2268DaE6c1421eb6B08e4e1dF525dA;
```

Comment:
It's used for the transfer too so I wouldn't call it STREAM IMO

Reply:
i struggled with this.  originally this was just the stream wallet but then Juan was afk and gov alpha said it was okay to transfer to this one too.  There is already a PL address in our wallets, so i need something more than `PHOENIX_LABS` and it _seemed_ the intent here was that this was the streaming wallet and transfers would/could go someplace else in the future.  Do you have a better name suggestion?

Reply:
I would just call it PHOENIX_LABS_2 or PHOENIX_LABS_WALLET since we already have PHOENIX_LABS in addresses_wallets